MachineLearningPrototype/dataCollection/collect_data.py

```diff
@@ -0,0 +1,170 @@
+import requests
+import argparse
+from datetime import datetime
+from datetime import date
+from math import ceil
+import time
+from pprint import pprint
+import pymongo
+
+def store_in_db(data, collection_name, db, primary_key='url'):
+	"""
+	Store data in db.collection_name (update if exists using primary key)
+	"""
+	db_col = db[collection_name]
+	record_id = {primary_key: data[primary_key]}
+	db_col.update_one(record_id, {"$set": data}, upsert=True)
+
+def store_on_fs(data, file_name):
+	"""
+	Store data in file named `file_name`
+	"""
+	with open(file_name, "w") as f:
+		f.write(data)
+
+def get_collection_records(collection_name, db):
```

Comment:
Did we use `get_collection_records()`?

Reply:
That was a helper function that I had written. Will remove it now. Thanks.